tests/metrics/test_data.py

```diff
@@ -34,21 +35,19 @@ def test_generate_repo_data(entropy_repository_paths: dict[str, pathlib.Path]) -
         assert data is not None
         assert isinstance(data, dict)
 
-        # Check for expected keys
-        expected_keys = [
-            "repo-path",
-            "repo-commits",
-            "repo-file-count",
-            "repo-commit-time-range",
-            "repo-includes-readme",
-            "repo-includes-contributing",
-            "repo-includes-code-of-conduct",
-            "repo-includes-license",
-            "repo-is-citable",
-            "repo-agg-info-entropy",
-            "repo-file-info-entropy",
-        ]
-        assert all(key in data for key in expected_keys)
+        # open the metrics table
+        with open(METRICS_TABLE, "r") as f:
+            metrics_table = yaml.safe_load(f)
```

Comment:
Cool to see these getting read from the metadata file! 😄

Reply:
Cheers thanks!